src/filters.js

```diff
@@ -30,8 +35,44 @@ const makeFilter = (
     values: Array.isArray(values) ? values : [values],
   });
 
-export default (params: any): FilterCreators => Object.keys(FILTERS).reduce(
-  (filters: FilterCreators, filter: string): FilterCreators =>
-    ({ ...filters, [filter]: makeFilter(FILTERS[filter], params) }),
-  {},
-);
+export const createFilter = (params: any): FilterCreators =>
+  Object.keys(FILTERS).reduce(
+    (filters: FilterCreators, filter: string): FilterCreators => ({
+      ...filters,
+      [filter]: makeFilter(FILTERS[filter], params),
+    }),
+    {},
+  );
+
+// todo make unit tests
+export const doesSatisfyToQueryFilters = (
+  item: Object,
+  queryFilters: Array<QueryFilter>,
+): boolean =>
+  queryFilters
+    .map((queryFilter: QueryFilter): boolean => {
```

Comment:
nice catch, but this should be `every` instead `map`, this way i can remove `every` from the bottom of the chain.

I don't like the name either, but i didn't come up with better. I'm using it here: 
https://github.com/Brewskey/brewskey.admin/pull/404/files#diff-ba2ba2eb26479a91c4fb8870f8861f5dR101
To check if newSocketItem pass all queryFilters of Entitytable, and if it does i increment counter for the table.